ocabundles.json

```diff
@@ -1,13 +1,21 @@
 {
-"63ZiwyeZeazA6AhYRYm2zD:2:Member Card:1.5.1": { "path": "OCABundles/schema/bcgov-digital-trust/member-card/OCABundle.json" },
-"Trx3R1frdEzbn34Sp1jyX:2:Member Card:1.5.1": { "path": "OCABundles/schema/bcgov-digital-trust/member-card/OCABundle.json" },
-"XUxBrVSALWHLeycAUhrNr9:2:Member Card:1.5.1": { "path": "OCABundles/schema/bcgov-digital-trust/member-card/OCABundle.json" },
-"63ZiwyeZeazA6AhYRYm2zD:2:Person:1.0": { "path": "OCABundles/schema/bcgov-digital-trust/showcase-lawyer-person/OCABundle.json" },
-"Trx3R1frdEzbn34Sp1jyX:2:Person:1.0": { "path": "OCABundles/schema/bcgov-digital-trust/showcase-lawyer-person/OCABundle.json" },
-"XUxBrVSALWHLeycAUhrNr9:2:Person:1.0": { "path": "OCABundles/schema/bcgov-digital-trust/showcase-lawyer-person/OCABundle.json" },
-"63ZiwyeZeazA6AhYRYm2zD:2:student_card:1.0": { "path": "OCABundles/schema/bcgov-digital-trust/student-card/OCABundle.json" },
-"Trx3R1frdEzbn34Sp1jyX:2:student_card:1.0": { "path": "OCABundles/schema/bcgov-digital-trust/student-card/OCABundle.json" },
-"XUxBrVSALWHLeycAUhrNr9:2:student_card:1.0": { "path": "OCABundles/schema/bcgov-digital-trust/student-card/OCABundle.json" },
+"AuJrigKQGRLJajKAebTgWu:3:CL:209526:default": { "path": "OCABundles/schema/bcgov-digital-trust/member-card/OCABundle.json" },
```

Comment:
Was this file updated manually or generated?  It should be generated, and the necessary changes made to the underlying source files.

Happy to have an automated way to generate this vs. it being explicitly added in each PR.  But it should not be manually updated without fixing the underlying data since the next person along will generate it and your changes will be lost.

Reply:
I manually generated this but I have updated all the underlying files in `/OCABundles` if that's what you're refering to

Reply:
Just pulled the PR and then ran the ` ./scripts/gen_ocabundlesjson.sh` script and I’m getting a different result than what has been committed in this PR for the 2 root JSON files.

This still needs to be fixed.

Reply:
okay got it, I was using the oca parser and the oca explorer to generate things, I must have missed the script in the docs. I'll get that fixed